packages/jaeger-ui/src/components/DeepDependencyGraph/getNodeLabel/DdgNode.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react';
+import cx from 'classnames';
+
+import focalNodeIcon from './focalNodeIcon';
+import setFocusButtonIcon from './setFocusButtonIcon';
+import NewWindowIcon from '../../common/NewWindowIcon';
+import { EViewModifier } from '../../../model/ddg/types';
+
+import './DdgNode.css';
+
+type TProps = {
```

Comment:
would it make sense for this to be `type TProps = TDdgVertex & { focalNodeUrl..., viewModifierStuff }`?

Reply:
The props don't quite line up:

- `vertexKey` vs `key` (which can't be named `key` as a prop)
- `label` is not present